### ISSUE TITLE
Testsuite Warning : DeprecationWarning

### ISSUE THREAD

**Issue by esarradj** [acoular/acoular#124]
CI runner warning:
/home/herold/anaconda3/envs/test_py37/lib/python3.7/site-packages/acoular-20.2-py3.7.egg/acoular/internal.py:14: DeprecationWarning: use "HasTraits.trait_get" instead
  vobj = list(vobj.get(i.rstrip('[]')).values())[0]

can we change vobj.get... to  vobj.trait_get... ?

**Comment by esarradj** [comment]
From traits.api
```
    # Defines the deprecated alias for 'trait_get'
    @deprecated('use "HasTraits.trait_get" instead')
    def get(self, *names, **metadata):
        return self.trait_get(*names, **metadata)

    get.__doc__ = trait_get.__doc__
```
Thus `trait_get` should work without a problem.

 *By Adam Kujawski on 2020-07-29T09:50:37 (imported from GitLab)*

**Comment by esarradj** [comment]
closed via commit b238ab38e758d2ec24193f0461314576bb3ceb2c

 *By Simon Jekosch on 2020-07-31T12:09:55 (imported from GitLab)*

**Comment by esarradj** [comment]
closed via merge request !131

 *By Simon Jekosch on 2020-07-31T12:09:55 (imported from GitLab)*
